tests/Unit/Core/ConstraintValidator/CleanHtmlWithIframeValidatorTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Core\ConstraintValidator;
+
+use PrestaShop\PrestaShop\Core\ConstraintValidator\CleanHtmlValidator;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\CleanHtml;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+class CleanHtmlWithIframeValidatorTest extends ConstraintValidatorTestCase
+{
+    public function testItFailsWhenScriptTagsAreGiven()
+    {
+        $scriptTag = '<script></script>';
```

Comment:
Just notice you create a new test class for it.
I recommend you to don't create class only for tests. Please put them into `CleanHtmlValidatorTest.php`, and add a dataProvider if needed :)

Reply:
The reason I created 2 test classes is so it works cleanly with class they extend: ConstraintValidatorTestCase. 
It's has function createValidator and then using whatever it returns for validation. Both of those tests require the same validator but different parameters.

So I think having 2 test classes is cleaner than hacking in the second validator in the same class.